src/backup/copy_job_async/copy_job_result.py

```diff
@@ -35,6 +36,16 @@ def source_bq_table(self):
         return BigQueryTable(self.source_project_id, self.source_dataset_id,
                              self.source_table_id)
 
+    @property
+    def source_table_reference(self):
+        table_id, partition_id = BigQueryTable \
```

Comment:
I would move this implementation to BigQueryTable class. Then you could have:
`return self.source_bq_table.get_table_reference()`